client/src/Components/local/atoms/core-mapping/helper/map-controls.tsx

```diff
@@ -30,9 +85,7 @@ const MapControls: React.FC<GeomanControlProps> = ({ onCreate, onChange }) => {
       snappingOption: true
     }}
     globalOptions={{}}
-    onCreate={e => onCreate(e as unknown as PM.ChangeEventHandler)}
-    onChange={e => onChange(e as unknown as PM.ChangeEventHandler)}
-
+    onCreate={e => map.removeLayer(e.layer)}
```

Comment:
Hello @lebaphi - I guess the above line is removing the new item from the GeoMan data since it is going to "arrive" via an updated `features` prop.  But, I _think_ we have to _wipe_ the GeoMan data on each `features` update - so it should not be necessary to `remove` the layer above.

Reply:
okay, I will remove this line